src/webportal/src/app/job/job-view/fabric/job-detail/conn.js

```diff
@@ -100,6 +100,31 @@ export async function fetchSshInfo() {
   }
 }
 
+export function setTensorBoardDisabled(jobInfo, rawJobConfig) {
+  return getTensorBoardUrl(jobInfo, rawJobConfig);
```

Comment:
why not use `getTensorBoardUrl` directly?

Reply:
Got it. Thanks.